src/main/java/com/rybka/Application.java

```diff
@@ -1,26 +1,45 @@
 package com.rybka;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.rybka.config.PropertyInfo;
 import com.rybka.dao.HibernateDAO;
 import com.rybka.service.BaseCurrencyExchangeConnector;
-import com.rybka.service.ExchangeService;
 import com.rybka.service.ExchangeRateConnector;
+import com.rybka.service.ExchangeService;
+import com.rybka.service.PrimeExchangeRateConnector;
 import com.rybka.view.ExchangeView;
 import coresearch.cvurl.io.request.CVurl;
 
+import java.net.http.HttpClient;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Scanner;
 
 public class Application {
+
     public static void main(String[] args) {
 
-        BaseCurrencyExchangeConnector baseCurrencyExchangeConnector = new ExchangeRateConnector(new CVurl());
+        var reader = new PropertyReader(PropertyInfo.PROPERTY_FILE_NAME).getProperties();
+        var currencyConnectorsMap = buildMap();
+        var connector = currencyConnectorsMap.get(reader.getProperty(PropertyInfo.EXCHANGE_SOURCE));
 
         ExchangeView view = new ExchangeView(
                 new Scanner(System.in),
-                new ExchangeService(baseCurrencyExchangeConnector),
+                new ExchangeService(connector),
                 new HibernateDAO());
 
         while (true) {
             view.showDialog();
         }
     }
+
+    public static Map<String, BaseCurrencyExchangeConnector> buildMap() {
+        return new HashMap<>() {{
+            put("PrimeExchangeRate", new PrimeExchangeRateConnector(
+                    HttpClient.newHttpClient(),
+                    new ObjectMapper()));
+            put("ExchangeRate", new ExchangeRateConnector(
```

Comment:
Property names could be constants or Enum

Reply:
fixed